<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QDialog" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>734</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Batch RAT Converter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <widget class="QListWidget" name="source_list">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QListWidget" name="target_list">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="button_remove_target">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Remove Target</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="button_remove_source">
     <property name="text">
      <string>Remove Source</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QPushButton" name="button_batch_convert">
     <property name="text">
      <string>Batch Convert</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_folders">
     <property name="text">
      <string>Source</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QProgressBar" name="progressbar">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>290</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>281</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>File Extensions</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QListWidget" name="bit_depth">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="add_extension">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="add_bit_depth">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QListView" name="list_extensions">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>290</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>431</width>
        <height>62</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>iConvert Path</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="path_iconvert">
         <property name="text">
          <string>C:/Program Files/Side Effects Software/Houdini 13.0.401/bin/iconvert.exe</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="button_iconvert_path">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>431</width>
        <height>56</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Destination Path</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLineEdit" name="destination_path">
         <property name="inputMethodHints">
          <set>Qt::ImhUrlCharactersOnly</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="button_destination_path">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Thread Count</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSpinBox" name="thread_count">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>source_list</zorder>
     <zorder>source_list</zorder>
     <zorder>source_list</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
   </item>
  </layout>
  <zorder>source_list</zorder>
  <zorder>button_batch_convert</zorder>
  <zorder>label_folders</zorder>
  <zorder>target_list</zorder>
  <zorder>button_remove_target</zorder>
  <zorder>button_remove_source</zorder>
  <zorder>progressbar</zorder>
  <zorder>label</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>groupBox_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
